File: Programas em C/01 - Basico - entrada e saida de dados/script6.c
/*
 *faça um algoritmo que pergunte quanto você ganha
 *por hora e o numero de horas trabalhadas no mês.
 *calcule e mostre o total do seu salario no referido mes
 */

#include <stdio.h>

int main(){
	//Declaração de variáveis
	float valor_por_hora, salario;
	int horas_trabalhadas;

	//Entradas
	printf("Qual valor você ganha por hora? ");
	scanf("%f", &valor_por_hora);

	printf("Quantas horas você trabalhou neste mês? ");
	scanf("%d", &horas_trabalhadas);

	//Processamento
	salario = (horas_trabalhadas * valor_por_hora);

	//Saída
	printf("Neste mês você vai receber R$ %.2f", salario);
}

File: Programas em Java/02 - If Else/Script6.java
/*
 * Elabore um algoritmo que leia as variaveis 'c' e 'n', respectivamente 
 * codigo e numero de horas trabalhadas de um operario. Calucle o salario 
 * sabendo-se que ele ganha R$10,00 por hora. QUando o numero de horas 
 * exceder a 50 calcule o excesso de pagamento armazenando-o na 
 * variavel 'e'. Caso contrario zerar tal variavel. A hora excedente de 
 * trabalho vale R$20,00. No final do processamento, imprimir o salario 
 * total e o salario excedente.
 */

package geek;

import java.util.Scanner;

public class Script6 {

	public static void main(String[] args) {
		//variáveis
		int c;
		float n, e = 0, salario, valor_hora = (float) 10.00;
		Scanner teclado = new Scanner(System.in);
		
		//Entradas
		System.out.println("Informe o código: ");
		c = teclado.nextInt();
		System.out.println("Informe a quantidade de horas trabalhadas: ");
		n = teclado.nextFloat();
		
		//Processamento
		if(n > 50) {
			e = (float)(n - 50) * (float)20.00;
			salario = (50 * valor_hora) + e;
			System.out.printf("Salário Total R$ %.2f\n", salario);
			System.out.printf("Salário excedente R$ %.2f\n", e);
		}else {
			salario = n * valor_hora;
			System.out.printf("Salário Total R$ %.2f\n", salario);
			System.out.printf("Salário excedente R$ %.2f\n", e);
		}
		teclado.close();
 
	}

}
